met-web/src/components/common/Modals/Schedule/index.tsx

```diff
@@ -23,7 +23,11 @@ const ScheduleModal = ({ reschedule, open, updateModal }: ScheduleModalProps) =>
     const [scheduledDate, setScheduledDate] = useState<Dayjs | null>(dayjs(Date.now()));
     const { savedEngagement, scheduleEngagement } = useContext(ActionContext);
     const dispatch = useAppDispatch();
-
+    const isValid =
```

Comment:
I think it should be called something like IsEngagementReady
but anyway can you put this in a separated validation method?

Reply:
one other. thing that I thought is.  that. we validate that to schedule the engagement, but the user. is still able to edit it after its been schedule, So in teory i can schedule the engagement and then remove the description, we might nneed to confirm with marie anik if this is okay